tests/test_updater_ng.py

```diff
@@ -109,14 +114,94 @@ def tearDown(self):
     # Logs stdout and stderr from the sever subprocess.
     self.server_process_handler.flush_log()
 
+  def _create_consistent_target(self, targetname: str, target_hash:str):
+    """Create consistent targets copies of their non-consistent counterparts
+    inside the repository directory.
+
+    Args:
+      targetname: A string denoting the name of the target file.
+      target_hash: A string denoting the hash of the target.
+
+    """
+    consistent_target_name = f"{target_hash}.{targetname}"
+    source_path = os.path.join(self.repository_directory, "targets", targetname)
+    destination_path = os.path.join(
+      self.repository_directory, "targets", consistent_target_name
+    )
+    shutil.copy(source_path, destination_path)
+
+
+  def _root_make_consistent_snapshot_true(self) -> bytes:
+    """Create a root Metadata instance, change roots consistent_snapshot to
+    true, self sign the changed root and then serialize the modified root back
+    to bytes."""
+    root_path = os.path.join(self.repository_directory, "metadata", "root.json")
+    root = Metadata.from_file(root_path)
+    root.signed.consistent_snapshot = True
+    root_key_path = os.path.join(self.keystore_directory, "root_key")
+    root_key_dict = import_rsa_privatekey_from_file(
+            root_key_path, password="password"
+    )
+    root_signer = SSlibSigner(root_key_dict)
+    root.sign(root_signer)
+    return root.to_bytes()
+
+
+  def test_refresh_on_consistent_targets(self):
+
+    # Make root consistent snapshot true.
+    root_modified_bytes = self._root_make_consistent_snapshot_true()
+    # Instantiate a new TrustedMetadataSet with changed root and modify internals.
+    # This seems better than modifying the actual root file in the client directory
+    # considering that the client directory is used from the rest of the tests.
+    trusted_metadata_set = TrustedMetadataSet(root_modified_bytes)
+    self.repository_updater._trusted_set = trusted_metadata_set
```

Comment:
I don't like tests modifying internal details.

I'm not sure I understand the comment here: both repository dir and client repo dir are set up in `setUp()` so it should be done for each test.